templates/zerver/home.html

```diff
@@ -73,12 +73,22 @@
       </div>
       <div id="empty_star_narrow_message" class="empty_feed_notice">
         <h4>{{ _("You haven't starred anything yet!") }}</h4>
+
+        {% trans %}
+        <p>Learn more about starring messages at
+        <a href="/help/star-a-message">/help/star-a-message</a>.</p>
```

Comment:
you'll want the full URL here, with the {{ realm_uri }}

Reply:
The part in `href` or the part that shows up? Assuming latter for now

Reply:
yup, the latter. Thanks!